search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -248,6 +249,59 @@ class TemporalEntityAttributeService(
             }
         }
 
+    @Transactional
+    suspend fun mergeAttribute(
```

Comment:
some of the logic here could be extracted and mutualized with the partial attribute operation (which is very similar... even identical with the exception of the default `observedAt`?)

Reply:
just gone through the spec and expected behaviors are actually not the same for partial attribute update (5.5.8) and merge entity with an existing attribute (5.5.12)